nightfall-client/src/event-handlers/block-proposed.mjs

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-cycle */
```

Comment:
:eyes: I can see this rule has been added to several files in the PR, is this needed? What was the error??

Reply:
This problem started to appear. I don't know exactly why.